src/xdocs/writingchecks.xml

```diff
@@ -86,37 +86,39 @@
 
       <p>
         Every Java Program is structured into files, and each of these
-        files has a certain structure. For example, if there is a
-        package statement then it is the first line of the file that is
-        not comment or whitespace. After the package statement comes a
-        list of import statements, which is followed by a class or
-        interface definition, and so on.
+        files has a certain structure. For example, a package statement
+        is always the first line of a file that is not comment or whitespace.
```

Comment:
This is not true, classes can be missing a package statement completely. It's usually not good practice, but it is allowed in a program. The original sentence had this clarification.

Reply:
Good point, fixed